profile/README.md

```diff
@@ -0,0 +1,33 @@
+[![](../assets/hcloud-banner-hero.jpg)](https://www.hetzner.com/cloud)
+
+**WELCOME TO THE INTERNET**
```

Comment:
Welcome to the Internet is now a duplicate, I think it would be better to put it into the alt text of the image.

```suggestion
[![Banner with the Hero mascot and the slogan "Welcome to the Internet"](../assets/hcloud-banner-hero.jpg)](https://www.hetzner.com/cloud)

**WELCOME TO THE INTERNET**
```